REFERENCE.md

```diff
@@ -21,35 +21,36 @@
 
 ### Defined types
 
-* [`nginx::resource::geo`](#nginxresourcegeo): Create a new geo mapping entry for NGINX
-* [`nginx::resource::location`](#nginxresourcelocation): Create a new location entry within a virtual host
-* [`nginx::resource::mailhost`](#nginxresourcemailhost): Define a mailhost
-* [`nginx::resource::map`](#nginxresourcemap): Create a new mapping entry for NGINX
-* [`nginx::resource::server`](#nginxresourceserver): Create a virtual host
-* [`nginx::resource::snippet`](#nginxresourcesnippet): Create a reusable config snippet that can be included by other resources
-* [`nginx::resource::streamhost`](#nginxresourcestreamhost): Create a virtual steamhost
-* [`nginx::resource::upstream`](#nginxresourceupstream): Create a new upstream proxy entry for NGINX
-* [`nginx::resource::upstream::member`](#nginxresourceupstreammember): Create an upstream member inside the upstream block.
+* [`nginx::resource::geo`](#nginx--resource--geo): Create a new geo mapping entry for NGINX
+* [`nginx::resource::location`](#nginx--resource--location): Create a new location entry within a virtual host
+* [`nginx::resource::mailhost`](#nginx--resource--mailhost): Define a mailhost
+* [`nginx::resource::map`](#nginx--resource--map): Create a new mapping entry for NGINX
+* [`nginx::resource::server`](#nginx--resource--server): Create a virtual host
+* [`nginx::resource::snippet`](#nginx--resource--snippet): Create a reusable config snippet that can be included by other resources
+* [`nginx::resource::streamhost`](#nginx--resource--streamhost): Create a virtual steamhost
+* [`nginx::resource::upstream`](#nginx--resource--upstream): Create a new upstream proxy entry for NGINX
+* [`nginx::resource::upstream::member`](#nginx--resource--upstream--member): Create an upstream member inside the upstream block.
 
 ### Data types
 
-* [`Nginx::DebugConnection`](#nginxdebugconnection)
```

Comment:
Most of REFERENCE.md appears to be rewritten, replacing all anchor references, maybe put this in a separate commit as this seems unrelated to the new type?

Reply:
done